_drafts/SC3-3-2-form-field-has-name.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Form field has accessible name
+description: |
+  Each form field element has an accessible name
+
+success_criterion:
+- 3.3.2 # Labels or Instructions
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rules applies to:
+- any element that is or [exposed to assistive technologies](#exposed-to-assistive-technologies),
```

Comment:
Shouldn't the applicability be restricted to form elements? Or is it to be applied to elements with the semantic role described below even if they are not used inside a form?

Reply:
@carlosapaduarte we don't want to restrict the rule to from elements as a form control could exist outside of a form element but would still need an accessible name.